### PR TITLE
Update location handling to 08022017 version

### PR DESCRIPTION
**Pull request by scop**
Signed-off-by: Ville Skyttä <ville.skytta@iki.fi>

### CODE REVIEW

**Comment by dnav** [comment]
Looks good.Thanks

File: examples/client/object_location.c [hunk 1]
```diff
@@ -56,26 +59,27 @@
...
 {
-    char     latitude   [5 + DEG_DECIMAL_PLACES]; //"359.12345" frag=5, 9+1=10! degrees +\0
-    char     longitude  [5 + DEG_DECIMAL_PLACES];
-    char     altitude   [5 + DEG_DECIMAL_PLACES];
-    char     uncertainty[5 + DEG_DECIMAL_PLACES];
+    float    latitude;
+    float    longitude;
+    float    altitude;
+    float    radius;
     uint8_t  velocity   [VELOCITY_OCTETS];        //3GPP notation 1st step: HORITZOL_VELOCITY_WITH_UNCERTAINTY
```
Comment (jvermillard) [inline examples/client/object_location.c]: not directly related but HORITZOL is a typo?

Comment (scop) [inline examples/client/object_location.c]: Yes, I noticed the HORITZOL typo and throught I'd address that in a separate PR, but I just added another commit fixing it to this one.

**Comment by scop** [comment]
One more uncertainty->radius change in this new revision, in the in the WITH_LOGS block.

**Pull request merged**